### PR TITLE
NOISSUE - Update dependencies

### PR DESCRIPTION
**Pull request by dborovcanin**
Signed-off-by: dusanb94 <dusan.borovcanin@mainflux.com>

### What does this do?
This pull request updates dependencies. It also fixes the warning in policies tests.

### Which issue(s) does this PR fix/relate to?
There is no such issue.

### List any changes that modify/break current functionality
There are no such changes.

### Have you included tests for your changes?
N/A

### Did you document any new/modified functionality?
N/A

### Notes
N/A

### CODE REVIEW

File: auth/api/http/policies/transport.go [hunk 1]
```diff
@@ -96,10 +96,11 @@ func encodeError(_ context.Context, err error, w http.ResponseWriter) {
 	default:
 		w.WriteHeader(http.StatusInternalServerError)
 	}
-	errorVal, ok := err.(errors.Error)
-	if ok {
-		if err := json.NewEncoder(w).Encode(errorRes{Err: errorVal.Msg()}); err != nil {
-			w.Header().Set("Content-Type", contentType)
+
+	errVal, ok := err.(errors.Error)
+	if ok && errVal.Msg() != "" {
```
Comment (drasko) [inline auth/api/http/policies/transport.go]: Maybe inline this initialization, if not used otherwise

File: auth/api/http/policies/responses.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ func (res deletePoliciesRes) Headers() map[string]string {
 }
 
 func (res deletePoliciesRes) Empty() bool {
-	return false
+	return true
```
Comment (drasko) [inline auth/api/http/policies/responses.go:42]: Why is this changed?

**Review by drasko — CHANGES_REQUESTED** [review]

File: auth/api/http/policies/responses.go (hunk 2 above)
Comment (dborovcanin) [reply auth/api/http/policies/responses.go:42]: Since `deleteResponse` has no body, `Empty` should return `false`.

Comment (drasko) [reply auth/api/http/policies/responses.go:42]: You introduced the change where it returns `true`. I am confused now - what should the return value be - `false` as it is today, or `true` as you propose.

Comment (dborovcanin) [reply auth/api/http/policies/responses.go:42]: My bad, it is empty, so it should return true.

Comment (drasko) [reply auth/api/http/policies/responses.go:42]: OK, good - resolving this one.

**Review by drasko — APPROVED** [review]
LGTM
